osu.Game/Overlays/Changelog/UpdateStreamBadge.cs

```diff
@@ -100,34 +100,41 @@ protected override void OnHoverLost(HoverLostEvent e)
 
         private void updateState()
         {
-            if (SelectedTab.Value == null && !externalDimRequested)
+            // highlighted regardless if we are hovered
+            bool textHighlighted = IsHovered;
+            bool barExpanded = IsHovered;
+
+            if (SelectedTab.Value == null)
             {
-                expandingBar.Expand();
-                expandingBar.FadeTo(1, transition_duration, Easing.OutQuint);
-                text.FadeTo(1, transition_duration, Easing.OutQuint);
-                return;
+                // at listing, all badges are highlighted when user is not hovering any badge.
+                textHighlighted |= !userHoveringArea;
+                barExpanded |= !userHoveringArea;
             }
+            else
+            {
+                // bar is always expanded when active
+                barExpanded |= Active.Value;
 
-            var shouldExpand = Active.Value || IsHovered;
-
-            expandingBar.IsCollapsed = !shouldExpand;
-            expandingBar.FadeTo(shouldExpand ? 1 : 0.5f, transition_duration, Easing.OutQuint);
+                // text is highlighted only when hovered or active (but not if in selection mode)
+                textHighlighted |= Active.Value && !userHoveringArea;
+            }
 
-            text.FadeTo(IsHovered || (Active.Value && !externalDimRequested) ? 1 : 0.5f, transition_duration, Easing.OutQuint);
+            expandingBar.Expanded = barExpanded;
+            text.FadeTo(textHighlighted ? 1 : 0.5f, transition_duration, Easing.OutQuint);
         }
 
-        private bool externalDimRequested;
+        private bool userHoveringArea;
 
-        public void EnableDim()
+        public bool UserHoveringArea
         {
-            externalDimRequested = true;
-            updateState();
-        }
+            set
+            {
+                if (value == userHoveringArea)
+                    return;
 
-        public void DisableDim()
-        {
-            externalDimRequested = false;
-            updateState();
+                userHoveringArea = value;
+                updateState();
+            }
         }
```

Comment:
Also could be a bindable?

Reply:
it could, but i didn't see much point. it's very local.